ros/src/tl_detector/tl_detector.py

```diff
@@ -276,14 +286,18 @@ def get_light_state(self):
 
```

Comment:
If `if(not self.has_image):` on line 283 is replaced by `if len(self.image_q) == 0:` then I think we can get rid of `self.has_image`.